.github/dependabot.yml

```diff
@@ -77,6 +60,8 @@ updates:
     allow:
       - dependency-name: Microsoft.SmartPlaces.Facilities.*
       - dependency-name: Mapped.*
+      - dependency-name: RealEstateCore.*
+      - dependency-name: WillowInc.*
```

Comment:
Looks like unrelated changes to the description.  Want all this in the same PR?

Reply:
Fair, I'll pull it out. Got carried away updating the builds